mailscanner/functions.php

```diff
@@ -247,7 +247,7 @@ function mailwatch_version()
 }
 
 if (!function_exists('imageantialias')) {
-    function imageantialias($image, $enabled)
```

Comment:
sure about this? if imageantialias is not present jpgraph will still pass 2 arguments to this function